runtime/jcl/common/com_ibm_jvm_Stats.c

```diff
@@ -78,11 +80,13 @@ Java_com_ibm_jvm_Stats_getStats(JNIEnv *env, jobject obj)
 		softmx = max;
 	}
 
-	/* Calculation of free physical memory from existing function in mgmtosext.c*/
-	free = Java_com_ibm_lang_management_internal_ExtendedOperatingSystemMXBeanImpl_getFreePhysicalMemorySizeImpl(env, obj);
 
-	/* Calculation of total physical memory from port lib api */
-	tot = (jlong) j9sysinfo_get_physical_memory();
+	rc = j9sysinfo_get_memory_info(&memInfo);
+	
+	if (0 == rc) { 
+		free = (jlong) memInfo.availPhysical;
```

Comment:
This doesn't have the same behavior as the original code, which sets free to -1 if the system call fails.
`(0 == rc)? (jlong) size: (jlong) -1`